canonical/global-nav
# update lxd logo and link target WD-4963

## Done
- Updated LXD logo
- Changed LXD link target

## QA
- Go to https://global-nav-261.demos.haus/
- Open the navigation
- Check the LXD logo and link target

## Issue
Fixes WD-4963

## Screenshot
![Screenshot from 2023-07-04 15-43-09](https://github.com/canonical/global-nav/assets/1155472/f449e5b6-5ab6-4445-bcdf-0d73144c9730)
